File: JAVA Solutions/CodeChef/GoodTurn.cpp
// GOOD TURN PROBLEM CODECHEF
#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int N,X,Y;
	cin>>N;
	while(N>0){
	cin>>X>>Y;
	    N--;
	if((X+Y)>6)
	    cout<<"YES\n";
	else
	    cout<<"NO\n";
	}
	return 0;
}

File: JAVA Solutions/CodeChef/GoodTurn.java
// GOOD TURN PROBLEM CODECHEF

import java.util.Scanner;

public class GoodTurn {
    public static void main(String[] args){
        Scanner read=new Scanner(System.in);
        int N=read.nextInt();
        while(N>0){
            int X=read.nextInt();
            int Y=read.nextInt();
            if((X+Y)>6){
                System.out.println("YES");
            }
            else{
                System.out.println("NO");
            }

        }
    }
}
